README.md

```diff
@@ -20,6 +20,8 @@ level of backwards compatibility to the official releases.
 
 ## Table of contents
 
+- [Releases and Versioning](#releases-and-versioning)
```

Comment:
I wouldn't have probably put this point as the first one of the readme, other that that it's ok

Reply:
Yeah I struggled with organization.. Feel free to modify if you have a better organization but it didn't seem to fit anywhere in the middle and it is too important to be at the end..

Reply:
I'll think about it :-)

Reply:
at the end of the day I think the way you put it it's right